contribs/gmf/less/icons.less

```diff
@@ -53,4 +53,8 @@
   &.gmf-icon-text:after {
     content: '\e906';
   }
+
+  &.gmf-icon-search-go:after {
+    content: '\e908';
```

Comment:
Indeed.
Let's be consistent for now and change all the selectors once for all.
Can you please open an issue for addressing this right after that?